azurerm/internal/services/apimanagement/tests/resource_arm_api_management_test.go

```diff
@@ -513,6 +519,12 @@ resource "azurerm_api_management" "test" {
       certificate          = filebase64("testdata/api_management_portal_test.pfx")
       certificate_password = "terraform"
     }
+
+    developer_portal {
+      host_name            = "developer-portal.terraform.io"
+      certificate          = filebase64("testdata/api_management_developer_portal_test.pfx")
+      certificate_password = "terraform"
+    }
```

Comment:
Please note the requirement of an additional hostname (please advise if it should be changed to a different one) and a certificate file in testdata/.